tests/index.json

```diff
@@ -1,15 +1,54 @@
-[
-  "welcome",
-  "power_of_puppet",
-  "resources",
-  "manifests_and_classes",
-  "modules",
-  "mysql",
-  "ntp",
-  "variables_and_parameters",
-  "conditional_statements",
-  "resource_ordering",
-  "defined_resource_types",
-  "agent_setup",
-  "application_orchestrator"
-]
+{
+  "welcome": { 
+    "watch_list": ["/root/.bash_history"],
+    "setup_command": "puppet apply ./manifests/welcome.pp"
+  },
+  "power_of_puppet": {
+    "watch_list": ["/root/.bash_history"],
+    "setup_command": "puppet apply ./manifests/welcome.pp"
```

Comment:
Yeah, it's just a place-holder for the moment. I suppose I could change it to an empty string or something.